oval_graph/client.py

```diff
@@ -176,9 +176,17 @@ def get_save_src(self, rule):
 
     def parse_arguments(self, args):
         self.prepare_parser()
+        self.prepare_parser_out()
         args = self.parser.parse_args(args)
         return args
 
+    def prepare_parser_out(self):
+        self.parser.add_argument(
+            '--output',
```

Comment:
Just a small suggestion - add here also `-o`, so one can specify output with minimum amount of writing.